src/core/coap/coap_secure.cpp

```diff
@@ -152,6 +152,35 @@ void CoapSecure::SetSslAuthMode(bool aVerifyPeerCertificate)
 
 #endif // OPENTHREAD_CONFIG_COAP_SECURE_API_ENABLE
 
+#if OPENTHREAD_CONFIG_COAP_BLOCKWISE_TRANSFER_ENABLE
+otError CoapSecure::SendMessage(Message &                   aMessage,
+                                ResponseHandler             aHandler,
+                                void *                      aContext,
+                                otCoapBlockwiseTransmitHook aTransmitHook,
+                                otCoapBlockwiseReceiveHook  aReceiveHook)
+{
+    otError error = OT_ERROR_NONE;
+
+    VerifyOrExit(IsConnected(), error = OT_ERROR_INVALID_STATE);
+
+    error = CoapBase::SendMessage(aMessage, mDtls.GetPeerAddress(), TxParameters::GetDefault(), aHandler, aContext,
```

Comment:
Rely on default parameter values to avoid special casing this?

Reply:
How would you suggest doing it? I'm a little lost on this one.

Reply:
Apologies, I misplaced this comment. Please ignore.